### DISCUSSION TITLE
Can we convert NSImage to SVG in Swift OSX ?

### DISCUSSION THREAD

**Discussion by YogeshPateliOS** [pocketsvg/PocketSVG#192] [Q&A]
Actually, I want to convert SFSymbol Image (as NSImage) to SVG. Thank you.

**Comment by arielelkin** [comment]
PocketSVG doesn't offer the functionality to convert NSImage to SVG.

SFSymbols are vector graphics, so you already have a lot of the functionality of an SVG. What are you trying to achieve?

**Reply by YogeshPateliOS** [reply]
Actually, I am making a MAC App that converts SFSymbol Image to SVG and Saving this to Download.
That's why I want to convert NSImage to SVG. Could you please help me out with this? 

Thanks!

**Reply by arielelkin** [reply]
But SF Symbols are a font, not a bitmap image. You may be confusing bitmaps with vector graphics?

**Reply by YogeshPateliOS** [reply]
Yes, I am a little bit confused about that. We are setting a system symbol so it's an image or not?
Any suggestion on how can I achieve this? Thanks!

**Reply by arielelkin** [reply]
Photos are bitmaps, so they're defined as a bidemensional array of pixels. Fonts and SVGs are [vector graphics](https://en.wikipedia.org/wiki/Vector_graphics), defined instead as a series of geometrical functions. 

Why do you want to convert a font to SVG?
